compiler/noirc_frontend/src/tests.rs

```diff
@@ -203,21 +208,33 @@ fn check_errors_with_options(src: &str, allow_parser_errors: bool, options: Fron
             continue;
         }
 
+        if let Some((span, message)) =
+            get_error_line_span_and_message(line, '-', byte, last_line_length)
+        {
+            custom_spans_with_errors.push((span, message));
+            continue;
+        }
```

Comment:
Unsure why we'd need to change this either. @asterite any insights?

Reply:
I think the overall location of the type alias was changed so these tests broke (though I'm not sure why we'd need custom spans here).

Reply:
I resolve the conversation because this has been reverted (after location change and update from master)